src/styles/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $padding-size;
+  gap: $gap-size;
+  width: calc(7 * #{$day-size} + 6 * #{$gap-size} + 2 * #{$padding-size});
+  box-sizing: border-box;
+
+  &__day {
+    width: $day-size;
+    height: $day-size;
+    background-color: $day-color;
+    border: 1px solid black;
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    transition-duration: 500ms;
```

Comment:
The `transition-duration` property is used without specifying which properties the transition is for. It's a good practice to specify the transition property to avoid unintended animations. For example, `transition: background-color 500ms, transform 500ms;`.